### PR TITLE
Fix test_tinderbox_scraper for ReadTimeout exceptions

### PR DESCRIPTION
**Pull request by whimboo**
This PR will fix issue #466 but is not ready yet. Lets see how the tests on travis will work.

### CODE REVIEW

**Comment by whimboo** [comment]
Dave it looks like that the test works way better when using pytest. I think we should get this landed. Does everything look fine?

**Comment by whimboo** [comment]
@mjzffr if you have a minute just to glance over the patch that would be nice. Not sure if @davehunt is around already. The changes are only pytest related so it should be fine for you to review. Thanks.

File: tests/remote/test_firefox.py [hunk 1]
```diff
@@ -4,318 +4,122 @@
...
+    {'branch': 'mozilla-central', 'platform': 'win32', 'locale': 'de'},
+    {'branch': 'mozilla-central', 'platform': 'win32', 'extension': 'txt'},
+    {'branch': 'mozilla-beta', 'platform': 'win32'},
+])
+def test_tinderbox_scraper(tmpdir, args):
+    """Test tinderbox scraper against the remote server."""
+    mozdownload.TinderboxScraper(destination=tmpdir, **args)
+
+
+@unittest.skip('Not testable as long as we cannot grab a latest build')
```
Comment (davehunt) [inline tests/remote/test_firefox.py]: No need for unittest, see https://docs.pytest.org/en/latest/skipping.html#skipping-test-functions or should we just delete the test?

File: tests/remote/test_firefox.py [hunk 2]
```diff
@@ -4,318 +4,122 @@
...
+def test_try_scraper():
+    pass
 
-    @unittest.skip('Not testable as long as we cannot grab a latest build')
-    def test_try_scraper(self):
-        pass
 
 if __name__ == '__main__':
     unittest.main()
```
Comment (davehunt) [inline tests/remote/test_firefox.py]: We can probably remove this if we're just discovering and running the tests using pytest.

File: tests/remote/test_firefox.py (hunk 1 above)
Comment (whimboo) [reply tests/remote/test_firefox.py]: Thanks. I filed PR #470 to cover both mentioned enhancements.

File: tests/remote/test_firefox.py (hunk 2 above)
Comment (whimboo) [reply tests/remote/test_firefox.py]: Good idea.

**Pull request merged**